.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     docker:
       # CircleCI maintains a library of pre-built images
       # documented at https://circleci.com/docs/2.0/circleci-images/
-      - image: circleci/python:3.8.4
+      - image: cimg/base:2021.05
```

Comment:
What is the difference here? A note of explanation would help.

Reply:
for newer versions of doyxgen, I will add a note for it.

Reply:
> I will add a note for it.

Still needed

Reply:
done

Reply:
Could this be made even clearer? i.e. The current version in the docker image. This will also ensure future changes note the new version.

Reply:
I tried to make it more clear, Is that okay?

Reply:
excellent, thanks.